src/app/_components/VoteForm/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ChangeEvent, useState } from 'react';
+
+import Icon from '@/src/components/Icon';
+import { cn } from '@/src/utils/cn';
+
+const VoteForm = () => {
+  const [options, setOptions] = useState<string[]>(['', '']);
```

Comment:
```suggestion
const LOW_LIMIT = 2;

  const [options, setOptions] = useState<string[]>(new Array(LOW_LIMIT).fill(''));
```

이런식으로 해주면 목적이더 잘드러나고, 상수 재사용도 쉬울것같은데 괜찮을까요?!

Reply:
고민해봤는데 그게 맞을 것 같네요 ! 반영하겠습니다. 
MAX_LIMIT도 상수로 해두는게 나을 것 같네요